packages/components/src/components/hds/accordion/types.ts

```diff
@@ -21,3 +21,15 @@ export enum HdsAccordionForceStateValues {
   Close = 'close',
 }
 export type HdsAccordionForceStates = `${HdsAccordionForceStateValues}`;
+
+export enum HdsAccordionItemTitleTagValues {
```

Comment:
@alex-ju I know we often define types like this on a per-component basis. Would it make sense to define these in one spot as HDS header types?

Reply:
good question. let's bring this to the engineering sync. there may be an opportunity to 'promote' some of our types at library level

Reply:
Added to the list of topics for a future eng sync!